borsh/src/lib.rs

```diff
@@ -94,7 +100,7 @@ compile_error!("feature \"std\" and feature \"hashbrown\" don't make sense at th
 
 #[cfg(all(feature = "schema", not(feature = "derive")))]
 compile_error!(
-    "feature \"schema\" depends on \"derive\" feature in its implementation; enable it too.."
+    "feature \"schema\" depends on \"derive\" feature in its implementation; enable it too"
 );
```

Comment:
Are there any limitation that blocks us from enabling it automatically from borsh Cargo.toml?